src/test/scala/wordbots/Randomizer.scala

```diff
@@ -0,0 +1,55 @@
+package wordbots
+
+import com.danielasfregola.randomdatagenerator.RandomDataGenerator
+import org.scalacheck.Gen.oneOf
+import org.scalacheck.rng.Seed
+import org.scalacheck.{Arbitrary, Gen}
+
+import scala.reflect.runtime.universe._
+import scala.util.{Random, Try}
+
+// NOTE: This takes a while to compile because of shapeless magic.
+object Randomizer extends RandomDataGenerator {
+  implicit lazy val arbInt: Arbitrary[Int] = Arbitrary(oneOf(0, 1, 2, 3))
+
+  import Semantics._
+
+  def action(): Option[Action] = safeRandom[Action]() match {
```

Comment:
Should these be called `generateRandomAction` and stuff? Seems like a weird name for a function to just call it `action`.

Reply:
Good idea, I'll change it.